src/domain/shared/components/ActivityLog/views/ActivityLogCardCommentCreatedView.tsx

```diff
@@ -7,7 +7,9 @@ export interface ActivityLogCardCommentCreatedViewProps extends ActivityLogViewP
 
 export const ActivityCardCommentCreatedView: FC<ActivityLogCardCommentCreatedViewProps> = props => {
   const { t } = useTranslation();
-  const action = t('components.activity-log-view.actions.card-comment-created');
+  const action = t('components.activity-log-view.actions.card-comment-created', {
+    parentdisplayName: 'cardDisplayName',
```

Comment:
Should be `{ parentDisplayName: props.parentDisplayName }`

Reply:
Fixed